tools/interop-node/feeder/hash_test.go

```diff
@@ -35,25 +35,25 @@ func Test_GeneratePrevoteHash(t *testing.T) {
 		{
 			name: "single chain",
 			args: args{
-				blockDataString: []string{"1:123:0x123"},
+				blockDataString: []string{"1/0x123/0x1"},
```

Comment:
How about changing variable name to `dataString`. FYI. The valid datastring format is "1/0x123/0x0:0x777"